backend/fms_core/services/sample.py

```diff
@@ -307,7 +308,7 @@ def extract_sample(process: Process,
 def prepare_library(process: Process,
                     sample_source: Sample,
                     container_destination: Container,
-                    library: Library,
+                    libraries_by_derived_sample,
```

Comment:
Add a docstring to the service to detail the dict structure (and the rest) to know what the function has to receive as input.